src/controller/datacontroller.js

```diff
@@ -182,8 +182,9 @@ export default class DataController {
 	 * used by e.g. collaborative editing plugin that syncs remote data on init.
 	 *
 	 * @fires init
-	 * @param {String} data Input data.
-	 * @param {String} [rootName='main'] Root name.
+	 * @param {String|Object.<String,String>} data Input data as a string or an object containing rootName - initialData pairs to
+	 * initialize data on multiple roots at once.
+	 * @param {String} [rootName='main'] Root name. Takes effect only if the first argument is provided as a string.
 	 * @returns {Promise} Promise that is resolved after the data is set on the editor.
 	 */
 	init( data, rootName = 'main' ) {
```

Comment:
I think `rootName = 'main'` does not make much sense now for this method. You can use the object if you want to init a different root.

Reply:
Yep, I was thinking about using object always in as the `data` parameter, but I think it would be too big breaking change. However, removing `rootName` should be fine because, as you said, most probably, no one uses it.

Reply:
Yep. See https://github.com/ckeditor/ckeditor5-editor-decoupled/issues/21#issuecomment-451898238.

Reply:
Actually https://github.com/ckeditor/ckeditor5-editor-decoupled/issues/21#issuecomment-451904627:)